### ISSUE TITLE
Move ammonia and hydrogen price to "Fuel prices"

### ISSUE THREAD

**Issue by mabijkerk** [quintel/etmodel#4227]
In the "Costs & efficiencies" section, the tab "Fuel prices" contains fossil fuels, nuclear fuels and biofuels (see screenshot below). I would expect the prices of imported hydrogen and ammonia to also be in this tab, but these are in the "Hydrogen" tab.
<img width="476" alt="Screenshot 2024-03-27 at 10 31 29" src="https://github.com/quintel/etmodel/assets/67338510/639e29fa-0553-4bf7-9084-0566272969d0">

@kaskranenburgQ @Charlottevm and maybe @AlexanderWirtz, what do you think: should the prices of hydrogen and ammonia be moved to the "Fuel prices" tab, under a new slide called "Hydrogen carriers", or is the current location more clear?

**Comment by kaskranenburgQ** [comment]
I agree! I think it would be better to move the hydrogen costs to this tab.

**Comment by kaskranenburgQ** [comment]
Isn't this issue also related to: https://github.com/quintel/etsource/issues/3260?
